frontend/src/components/CreateBattle.svelte

```diff
@@ -73,6 +75,17 @@
             })
     }
 
+    function getPossiblePointValues() {
```

Comment:
Instead of adding api calls for config values, my preferred solution would be to add an appConfig object in the index HTML, which would be available through a global javascript variable of appConfig.  This can include multiple values, and saves making api calls to render the pages.

Reply:
e.g. in frontend/public/index.html
`<script>
appConfig = {{ .AppConfig }}
</script>`

and in handleIndex() add an AppConfig struct and jsonMarshal (go's json.stringify equivalent) it, passing it as apart of the UIConfig struct.

Reply:
Ah... yes, you're right, that sounds much better.  I will fix.